functions/exercise2/scriptSpec.js

```diff
@@ -1 +1,35 @@
 /// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals isPalindrome*/
+
+describe("Checking if word is palindrome: ", function () {
+    it("word 'tenet' is a palindrome", function () {
+        var palindrome = isPalindrome("tenet");
+        expect(palindrome).toBeTruthy();
+    });
+    it("word dog isn't a palindrome", function () {
+        var palindrome = isPalindrome("dog");
+        expect(palindrome).toBeFalsy();
+    });
+    it("if after palidromem is spacia 'kayak ', it is palidrom too", function () {
+        var palindrome = isPalindrome("kayak ");
+        expect(palindrome).toBeTruthy();
+    });
+    it("if before palidromem is spacia ' kayak', it is palidrom too", function () {
+        var palindrome = isPalindrome(" kayak");
+        expect(palindrome).toBeTruthy();
+    });
+    it("if between two palindromes is spacia 'kayak kayak', it is a palindrome too", function () {
+        var palindrome = isPalindrome("kayak kayak");
+        expect(palindrome).toBeTruthy();
+    });
+    it("One letter, 'a' is palindrome", function () {
+        var palindrome = isPalindrome("a");
+        expect(palindrome).toBeTruthy();
+    });
+    it("no letters, '' is not a palindrome", function () {
```

Comment:
Description is incorrect. On the basis of the description you may asses that function will return false for empty string.
But it does not return false - it throws exception - it should be specified in test description.